<launch>
  <node pkg="mechanism_control" type="spawner.py" args="$(find pr2_default_controllers)/laser_tilt_controller.xml" output="screen"/>
  <node pkg="pr2_mechanism_controllers" type="control_laser.py" 
args="laser_tilt_controller sine 3 0.78 0.3" />
</launch>
